javascript/ql/test/library-tests/ModuleImportNodes/TSGlobalImport/client1.ts

```diff
@@ -0,0 +1,6 @@
+// <reference path="framework1.d.ts" />
```

Comment:
No need to create a separate folder; just add these tests to `javascript/ql/test/library-tests/ModuleImportNodes`.